enacts/wat_bal/agronomy.py

```diff
@@ -214,3 +214,109 @@ def soil_plant_water_balance(
             - (planted_since - np.timedelta64(1, "D"))
         )
     return sm, drainage, et_crop, planting_date
+
+
+def weekly_api_runoff(
+    daily_rain,
+    no_runoff=12.5,
+    api_thresh=xr.DataArray([6.3, 19, 31.7, 44.4, 57.1, 69.8], dims=["api_cat"]),
+    api_poly=xr.DataArray(
+        [
+            [0.858, 0.0895, 0.0028],
+            [-1.14, 0.042, 0.0026],
+            [-2.34, 0.12, 0.0026],
+            [-2.36, 0.19, 0.0026],
+            [-2.78, 0.25, 0.0026],
+            [-3.17, 0.32, 0.0024],
+            [-4.21, 0.438, 0.0018],
+        ],
+        dims=["api_cat", "powers"],
+    ),
```

Comment:
api_thresh and api_poly here are mutable defaults, which are something [we should avoid](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). Do this instead:
 ```
def weekly_api_runoff(
    ...
    api_thresh=None,
    ....
):
    if api_thresh is None:
        api_thresh = DEFAULT_API_THRESH.copy(deep=True)
```

Reply:
Is that irrelevant now with the new setup?